src/config.rs

```diff
@@ -431,9 +431,26 @@ pub enum Config {
     /// used for signatures, encryption to self and included in `Autocrypt` header.
     KeyId,
 
-    /// This key is sent to the self_reporting bot so that the bot can recognize the user
+    /// Send statistics to Delta Chat's developers.
+    /// Can be exposed to the user as a setting.
+    SendStatistics,
```

Comment:
Maybe prefix everything with `Stats` then? E.g. `StatsSending, StatsLastSent, StatsId, StatsLastMsgId, StatsLastContactId`. I think it's better w/o "Excluded" and "Old" because even the comment says "included". Alternatively, replace both words with "Included" or "Counted", these sound better to me.